### PR TITLE
setup ui test framework

### LINKED ISSUE #23: [Test]Setup UI autotest framework
Autotest framework:
python3+selenium2

### PR DESCRIPTION
**Pull request by gyfpython**
resolve #23

### CODE REVIEW

File: test/Smoking_test_suite/utils/logfuction.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    }#日志级别关系映射
+
+    def __init__(self,filename="../log/check.log",level='info',when='D',backCount=3,fmt='%(asctime)s - %(pathname)s[line:%(lineno)d] - %(levelname)s: %(message)s'):
+        self.logger = logging.getLogger(filename)
+        format_str = logging.Formatter(fmt)#设置日志格式
+        self.logger.setLevel(self.level_relations.get(level))#设置日志级别
+        sh = logging.StreamHandler()#往屏幕上输出
+        sh.setFormatter(format_str) #设置屏幕上显示的格式
+        th = handlers.TimedRotatingFileHandler(filename=filename,when=when,backupCount=backCount,encoding='utf-8')#往文件里写入#指定间隔时间自动生成文件的处理器
+        #实例化TimedRotatingFileHandler
```
Comment (xiayy860612) [inline test/Smoking_test_suite/utils/logfuction.py]: rm unusage code, if any help knowledge, put into wiki.

**Review by xiayy860612 — CHANGES_REQUESTED** [review]
rm all binay files, you can add a script to intall some dependences.

File: test/Smoking_test_suite/testcases/testtitle.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# coding=utf-8
+from selenium import webdriver
+from page.homePage import HomePage
+from config.environment import *
+from selenium.webdriver.common.by import By
+import unittest
+
+class gettitle(unittest.TestCase):
+    def setUp(self):
+        self.driver = webdriver.Chrome('../resources/chromedriver.exe')
```
Comment (xiayy860612) [inline test/Smoking_test_suite/testcases/testtitle.py]: how about linux?

**Review by xiayy860612 — CHANGES_REQUESTED** [review]

File: test/Smoking_test_suite/utils/downloaddriver.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        with open('../resources/chromedriver.zip', "wb") as file:
+            for data in response.iter_content(chunk_size=chunk_size):
+                file.write(data)
+    unzip("../resources/chromedriver.zip", "../resources/")
+    os.remove("../resources/chromedriver.zip")
+
+def downloadDriver():
+    os_sys = checkos()
+    if os_sys.lower() == 'windows':
+        download("http://npm.taobao.org/mirrors/chromedriver/73.0.3683.68/chromedriver_win32.zip")
```
Comment (xiayy860612) [inline test/Smoking_test_suite/utils/downloaddriver.py]: I'm not sure if we can use pip install this chromedriver.
refer to https://pypi.org/project/chromedriver-binary/

File: test/Smoking_test_suite/requirement.txt [hunk 4]
```diff
@@ -1,2 +1,3 @@
 selenium==3.141.0
-html-testRunner==1.2
\ No newline at end of file
+html-testRunner==1.2
+requests
```
Comment (xiayy860612) [inline test/Smoking_test_suite/requirement.txt]: you'd better add version for requests library.

File: test/Smoking_test_suite/utils/downloaddriver.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+def unzip(sou,dst):
+    r = zipfile.is_zipfile(sou)
+    if r:
+        fz = zipfile.ZipFile(sou, 'r')
+        for file in fz.namelist():
+            fz.extract(file, dst)
+
+
+def download(downloadurl):
```
Comment (xiayy860612) [inline test/Smoking_test_suite/utils/downloaddriver.py]: it seems an private method, you'd better add prefix `__`  for method name

**Review by xiayy860612 — CHANGES_REQUESTED** [review]
merge all changes into one commit:

- git fetch
- git rebase -i orgin dev, in edit first one should be `p`, others use `s` for each commit
- git push -f

**Comment by gyfpython** [comment]
git rebase 4 commits to 1/requests version added/prefix __ for private method name

**Review by xiayy860612 — APPROVED** [review]

**Pull request merged**